### ISSUE TITLE
Exam list

### ISSUE THREAD

**Issue by zcjess** [zcjess/Ecoach-accessibility-issues#7]
,WCAG 1.3.1,WCAG 4.1.2,Best Practice: Ensures role attribute has an appropriate value for the element (#categoryList) #7

#### Issue
`Ensures role attribute has an appropriate value for the element` ([aria-allowed-role](https://dequeuniversity.com/rules/axe/4.6/aria-allowed-role?application=msftAI))
#### Target application
[ECoach | STATS250 Grade Calculator](https://umecoach.miserver.it.umich.edu/gradebook/coach/1/GradeCalculator/)
#### Element path
#categoryList
#### Snippet
`<ul id="categoryList" role="region" aria-live="polite" class="categories">`
#### How to fix
    Fix any of the following:
      ARIA role region is not allowed for given element
#### Environment
Chrome version 110.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
<img width="798" alt="image" src="https://user-images.githubusercontent.com/90011962/223481049-b7520510-03a8-434b-8764-617db56ca11e.png">

**Comment by zcjess** [comment]
WCAG 1.3.1: Ensures <li> elements are used semantically (.category-item.grade-display[aria-controls="assignmentCategory1"])

#### Issue
`Ensures <li> elements are used semantically` ([listitem](https://accessibilityinsights.io/info-examples/web/listitem))
#### Target application
[ECoach | STATS250 Grade Calculator](https://umecoach.miserver.it.umich.edu/gradebook/coach/1/GradeCalculator/)
#### Element path
.category-item.grade-display[aria-controls="assignmentCategory1"]
#### Snippet
`<li aria-controls="assignmentCategory1" class="category-item grade-display item1"><span class="color-indicator"></span><span class="category-name">Exam:</span><span class="category-percent">0%</span><span>|</span><span class="category-weight">40% of tot...`
#### How to fix
    Fix any of the following:
      List item does not have a <ul>, <ol> parent element without a role, or a role="list"
#### Environment
Chrome version 110.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
